src/functional/ffront/type_info.py

```diff
@@ -514,11 +544,11 @@ def function_signature_incompatibilities_fieldop(
 def function_signature_incompatibilities_scanop(
     scanop_type: ct.ScanOperatorType, args: list[ct.SymbolType], kwargs: dict[str, ct.SymbolType]
 ) -> Iterator[str]:
-    if not all(isinstance(arg, ct.FieldType) for arg in args):
-        yield "Arguments to scan operator must be fields."
+    if not all(is_field_type_or_tuple_of_field_type(arg) for arg in args):
+        yield "Arguments to scan operator must be fields or tuples thereof."
```

Comment:
```suggestion
        yield "Arguments to scan operator must be either fields or tuples."
```

Reply:
The *thereof* is important to signify that the tuple must contain fields. I don't think the *either* is necessary and makes it only more verbose. If you think it's easier to understand with the either I can add it.

Reply:
it is really not a big deal :)